pkg/controller/provider/model/vsphere/tree.go

```diff
@@ -79,7 +86,14 @@ func (r *Tree) Build() (*TreeNode, error) {
 			folder := model.(*Folder)
 			refList := RefList{}
 			refList.With(folder.Children)
+		next:
 			for _, ref := range refList {
```

Comment:
I only see one `for` here, is the label on the continue necessary?

Reply:
Yes because a _continue_ (or break) in Go applies to the inner most loop/switch so a _naked_ continue would only apply to the switch but I needed it to apply to the `for` loop.